modules/claims_api/spec/lib/claims_api/v2/disability_compensation_validation_spec.rb

```diff
@@ -53,6 +53,57 @@ class TestDisabilityCompensationValidationClass
     end
   end
 
+  describe '#validate_form_526_location_codes' do
```

Comment:
Again, I wonder if we tested an invalid code whether `validate_form_526_location_codes` would catch it (I suspect not). To explore in a future task.

Reply:
Correct, at the moment I do not think it would in our validation, although I do feel like I have seen this error, but I'm wondering if I saw it coming back from the docker container or something, just not sure.